### PR TITLE
Crusher Balance

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request
Crusher Code slightly cleaner
Crushers do more damage

## Why It's Good For The Game
I want to see how they feel like this.

## Changelog

:cl:
balance: Crusher damage has been slightly increased across the board.
/:cl:

### CODE REVIEW

**Comment by Sun-Soaked** [comment]
transforming weapons have been refactored, this pr will probably need to be adjusted

File: code/modules/mining/equipment/kinetic_crusher.dm [hunk 1]
```diff
@@ -28,11 +28,13 @@
...
 	var/detonation_damage = 20
 	var/backstab_bonus = 10
+	var/unwielded_force = 0
+	var/wielded_force = 25
 
 /obj/item/kinetic_crusher/ComponentInitialize()
 	. = ..()
 	AddComponent(/datum/component/butchering, 60, 110) //technically it's huge and bulky, but this provides an incentive to use it
-	AddComponent(/datum/component/two_handed, force_unwielded=0, force_wielded=15)
+	AddComponent(/datum/component/two_handed, unwielded_force, wielded_force)
```
Comment (SomeguyManperson) [inline code/modules/mining/equipment/kinetic_crusher.dm]:
```suggestion
	AddComponent(/datum/component/two_handed, force_unwielded=unwielded_force, force_wielded=wielded_force)
```

**Comment by thgvr** [comment]
canm we just remove this thing already

**Comment by Sun-Soaked** [comment]
> canm we just remove this thing already

wyci

**Review by Sun-Soaked — APPROVED** [review]
another syndicate crusher sterilized

**Pull request merged**